metaflow/plugins/kfp/kfp_step_decorator.py

```diff
@@ -0,0 +1,36 @@
+from metaflow.decorators import StepDecorator
+from metaflow.exception import MetaflowException
+
+
+class KfpStepDecorator(StepDecorator):
+    """
+    For KFP orchestrator plugin only.
+
+    Step decorator to specify the image for the step.
+
+    To use, follow the example below.
+    ```
+    @kfp_step(image="image_on_dockerhub")
+    @step
+    def start(self):
+        self.next(self.next_step)
+
+    ```
+
+    Parameters
+    ----------
+    image: str
+        Defaults to None.
+        Must resolve to an actual image, either publicly hosted
+        or available for download on the customer's infra.
+    """
+
+    name = 'kfp_step'
+
+    defaults = {
+        "image": None,
+    }
```

Comment:
Does this Flow work when running locally? Like in [test_example_flow.py](http://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-examples/aip-workflow-example/blob/1fd6dd6576f88e5a5f3c9f62d3260b3b76e4351b/aip_workflow_example/tests/test_example_flow.py#L23-L23)

Reply:
It does run locally.